fix(docs): tutorials tags cleanup final batch (#3318)
scaleway/docs-content: tutorials/s3-customize-url-cname/index.mdx

- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
- Access to the [DNS zone](/network/domains-and-dns/concepts/#dns-zone) of your domain name

> 1. From the Scaleway console, [create a new bucket](/storage/object/how-to/create-a-bucket/). When entering a name for your bucket, the name must be the same as the domain/subdomain intended to be used with it. For example, to create a bucket accessible at `download.example.com` the bucket name must be `download.example.com`.

Reviewer: ```suggestion
1. From the Scaleway console, [create a new bucket](/storage/object/how-to/create-a-bucket/). When entering a name for your bucket, the name must be the same as the domain/subdomain you intend to use with the bucket. For example, to create a bucket accessible at `download.example.com` the bucket name must be `download.example.com`.
```
